components/raftstore/src/store/fsm/store.rs

```diff
@@ -1408,9 +1409,6 @@ impl<EK: KvEngine, ER: RaftEngine> RaftBatchSystem<EK, ER> {
         MEMTRACE_RAFT_ROUTER_ALIVE.trace(TraceEvent::Reset(0));
         MEMTRACE_RAFT_ROUTER_LEAK.trace(TraceEvent::Reset(0));
 
-        if let Some(h) = handle {
```

Comment:
Does this change the behavior of `shutdown`?

Reply:
It does change the shutdown of pd worker from asynchronously to synchronously, but I think this doesn't have other side effects.